swablu/discord_util.py

```diff
@@ -15,7 +15,8 @@ async def regenerate_message(discord_client: Client, channel_id: int, message_id
         message_id = message.id
     else:
         message = await channel.fetch_message(message_id)
-        await message.edit(content=text)
+        if message.content != text:
+            await message.edit(content=text)
```

Comment:
I'm a little fuzzy on python. In `message.edit(content=text)`, what is `content`? Are we assigning to a variable then throwing it away? We're probably not passing in a boolean since iirc python uses `==` for comparison.

Reply:
I see, we're skipping whatever the `*` parameter is. https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.edit

Reply:
`content` is the name of one of the formal parameters of `message.edit()`. Here we set that specific parameter to the string contained in `text`.